packages/flutter/lib/src/widgets/overscroll_indicator.dart

```diff
@@ -849,7 +849,7 @@ class _StretchController extends ChangeNotifier {
     velocity = clampDouble(velocity, 1, 10000);
     _stretchSizeTween.begin = _stretchSize.value;
     _stretchSizeTween.end = math.min(_stretchIntensity + (_flingFriction / velocity), 1.0);
-    _stretchController.duration = Duration(milliseconds: (velocity * 0.02).round());
+    _stretchController.duration = Duration(milliseconds: math.max(velocity * 0.02, 50).round());
```

Comment:
we should probably define a const in this class
something like
```dart
const double _kMinStretchDuration = 50;
```

Reply:
@gilnobrega can you follow up here?

Reply:
I created constants for the minimum and maximum values of `velocity` and the minimum value of stretch duration.

I opted not to name them starting with a `k` prefix because other constants in the class do not have it, and they are not global constants.